nncf/quantization/api/engine.py

```diff
@@ -12,17 +12,15 @@
 """
 
 from abc import ABC, abstractmethod
-from typing import Callable, Dict, TypeVar
+from typing import Callable, TypeVar, Dict
```

Comment:
What do you think about removing the `api` folder and putting this file directly in the `nncf/quantization/`?

Reply:
Good suggestion. Since `api` folder contains only the `engine.py` file, we can move it into `nncf/quantization` and add it into `__init__.py` nearby the `quantize` method. @AlexKoff88, @alexsu52, what do you think?

Reply:
I would prefer to move engine.py into nncf directly because this abstraction is agnostic to algorithms and should be implemented in the each backend.